examples/streaming.js

```diff
@@ -0,0 +1,14 @@
+const { Unleash } = require('../lib');
+
+const client = new Unleash({
+  appName: 'my-application',
+  url: 'https://app.unleash-hosted.com/demo/api/',
+  customHeaders: {
+    Authorization: '943ca9171e2c884c545c5d82417a655fb77cec970cc3b78a8ff87f4406b495d0',
+  },
+  experimentalMode: { type: 'streaming' },
```

Comment:
this has to be provided explicitly to opt into streaming. The reason I model it as {type: 'polling'} | {type: 'streaming'} union is to add streaming config option later {type: 'streaming', config: {refresh, backoff}}